Communication guide: overview, blogs, podcast, social media, presentation, lay summary chapters (#1929)
alan-turing-institute/the-turing-way: book/website/communication/comms-overview/comms-overview-resources.md

# Resources and Recommendations

(cm-comms-overview-resources-accessibility)=
## Accessibility

Read more in this chapter: {ref}`cm-comms-overview-accessibly`.

* [Ability Net](https://abilitynet.org.uk/) for guides to making your documents more accessible.
* [Contrast calculator](https://contrastchecker.com/) is a good website to freely check if you meet the Web Content Accessibility Guidelines (WCAG) ratings.
* [Outlier conf](https://www.outlierconf.com/) is a good example of accessibility in conference participation fees and data experience (vs visualisation).

(cm-comms-overview-resources-blogs)=
## Blogs

Read more in this chapter: {ref}`cm-blogs`.

* [20 metrics every blogger needs to track to measure success](https://www.dreamhost.com/blog/metrics-every-blogger-needs-to-track/).
* [Start a Wordpress blog](https://www.podcastinsights.com/start-a-wordpress-blog/)
* [Comparing Google sites and Wordpress](https://superbwebsitebuilders.com/google-sites-vs-wordpress)

### Some of our favourite blogs
* Dr Veronika Cheplygina's [How I fail series](https://veronikach.com/failure/)
* [Software Sustainability Institute Blog](https://www.software.ac.uk/blog)

> * [Open Working](https://openworking.wordpress.com/) by from 4TU.ResearchData & TU Delft Library. Uses WordPress.

Reviewer: ```suggestion
* [Open Working](https://openworking.wordpress.com/) by 4TU.ResearchData & TU Delft Library and Data Stewards. Uses WordPress.
```

* [Metadocencia](https://metadocencia.netlify.app/post/) - This is in Spanish but you can also read it in English [here](https://metadocencia.netlify.app/en/post/).

(cm-comms-overview-resources-lay)=
## Lay summaries

Read more in this chapter: {ref}`cm-lay-summaries`.

> * Good example of how abstract can be changed into a lay summary - [Perfecting that lay summary](https://bitesizebio.com/10871/perfecting-that-lay-summary/).

Reviewer: ```suggestion
* See  [Perfecting that lay summary](https://bitesizebio.com/10871/perfecting-that-lay-summary/) for a good example of how an abstract can be changed into a lay summary.
```

### Some of our favourite podcasts
* [Orion Open Science](https://www.orion-openscience.eu/publications/training-materials/201902/podcasts).
* [Open Science talk](https://soundcloud.com/opensciencetalk).
* [The Turing Podcast](https://www.turing.ac.uk/news/turing-podcast).
* [ReproducibiliTea Podcast](https://soundcloud.com/reproducibilitea).
* [Open science stories](https://podcasts.apple.com/gb/podcast/open-science-stories/id1547403532).

(cm-comms-overview-resources-presentations)=
## Presentations

Read more in this chapter: {ref}`cm-presentations`.

Here are some websites that offer free resources making different aspects of presenations:

* [The Noun Project](https://thenounproject.com) for icons and photographs.
* [Unsplash](https://unsplash.com) for photographs.
* [Pexels](https://www.pexels.com) for photographs.
* [Freepik](https://www.freepik.com/free-photos-vectors/english) for icons and illustrations.
* [Flaticon](https://www.flaticon.com/free-icons/english) for icons.
* [Canva](https://www.canva.com) for icons.

For recording talks:
* [OBS](https://obsproject.com).

> * See [this guide for using OBS](https://software.ac.uk/fellowship-programme/2019/application-video-guide) by the Software Sustainability Institute.

Reviewer: ```suggestion
See [this guide for using OBS](https://software.ac.uk/fellowship-programme/2019/application-video-guide) by the Software Sustainability Institute.
```

> * [Amara](https://amara.org) or [Otter](https://otter.ai/) can be used to generate your subtitles. Video conferencing platforms such as Zoom or Google meet also have captions.

Reviewer: ```suggestion
* [Amara](https://amara.org) or [Otter](https://otter.ai/) can be used to generate your subtitles. 
Video conferencing platforms such as Zoom or Google meet also have captions.
```
